projects/packages/account-protection/src/class-account-protection.php

```diff
@@ -83,6 +105,21 @@ protected function register_runtime_hooks(): void {
 
 		// Add password detection flow
 		add_action( 'login_form_password-detection', array( $this->password_detection, 'render_page' ), 10, 2 );
+		add_action( 'wp_enqueue_scripts', array( $this->password_detection, 'enqueue_styles' ) );
+
+		// Add password validation
+		add_action(
+			'user_new_form',
+			function () {
+				wp_nonce_field( 'add-new-user', '_new_user_nonce' );
```

Comment:
The core `user_new_form` has an existing `_wpnonce_create-user` nonce, if we need to add our own custom nonce here maybe a prefixed name like `_jetpack_nonce_create-user` to avoid future confusion?

<img width="447" alt="Screenshot 2025-02-05 at 12 01 34 PM" src="https://github.com/user-attachments/assets/6a511f49-267d-4e8f-8319-d6aba5cd12aa" />

Reply:
Nice catch - missed that that was there entirely!

Reply:
Done in [b651b2b](https://github.com/Automattic/jetpack/pull/41401/commits/b651b2b6a06271044f7c03fc1766472d65cc54e7)